### DISCUSSION TITLE
What event is triggered when the calendar need new events data?

### DISCUSSION THREAD

**Discussion by stescacom** [vkurko/calendar#223] [Q&A]
Hi,

If we want to navigate through the calendar the next and previous button do not trigger any event. However, it is mentioned in the documentation for `eventSources` that the request is triggered when the calendar need data; I want to know which event is triggered for it?

**Comment by vkurko** [comment]
Hello,

The next and previous buttons trigger [datesSet](https://github.com/vkurko/calendar?tab=readme-ov-file#datesset) event.
